multiregion/docs/multiregion.rst

```diff
@@ -447,11 +534,87 @@ There is a script you can run to collect the JMX metrics from the command line,
+
+      ==> Describe topic multi-region-async
+
+      Topic: multi-region-async	PartitionCount: 1	ReplicationFactor: 4	Configs: min.insync.replicas=1,confluent.placement.constraints={"version":1,"replicas":[{"count":2,"constraints":{"rack":"west"}}],"observers":[{"count":2,"constraints":{"rack":"east"}}]}
+      	Topic: multi-region-async	Partition: 0	Leader: 2	Replicas: 1,2,4,3	Isr: 2	Offline: 1	Observers: 4,3
+
+      ==> Describe topic multi-region-default
+
+      Topic: multi-region-default	PartitionCount: 1	ReplicationFactor: 4	Configs: min.insync.replicas=1,confluent.placement.constraints={"version":1,"replicas":[{"count":2,"constraints":{"rack":"west"}}],"observers":[{"count":2,"constraints":{"rack":"east"}}]}
+      	Topic: multi-region-default	Partition: 0	Leader: 2	Replicas: 1,2,3,4	Isr: 2	Offline: 1	Observers: 3,4
+
+      ==> Describe topic under-min-isr-promotion
+
+      Topic: under-min-isr-promotion	PartitionCount: 1	ReplicationFactor: 4	Configs: min.insync.replicas=2,confluent.placement.constraints={"observerPromotionPolicy":"under-min-isr","version":2,"replicas":[{"count":2,"constraints":{"rack":"west"}}],"observers":[{"count":2,"constraints":{"rack":"east"}}]}
+      	Topic: under-min-isr-promotion	Partition: 0	Leader: 2	Replicas: 2,1,3,4	Isr: 2,4	Offline: 1	Observers: 3,4
+
+      ==> Describe topic under-replicated-promotion
+
+      Topic: under-replicated-promotion	PartitionCount: 1	ReplicationFactor: 4	Configs: min.insync.replicas=1,confluent.placement.constraints={"observerPromotionPolicy":"under-replicated","version":2,"replicas":[{"count":2,"constraints":{"rack":"west"}}],"observers":[{"count":2,"constraints":{"rack":"east"}}]}
+      	Topic: under-replicated-promotion	Partition: 0	Leader: 2	Replicas: 2,1,3,4	Isr: 2,4	Offline: 1	Observers: 3,4
+
+      ==> Describe topic leader-is-observer-promotion
+
+      Topic: leader-is-observer-promotion	PartitionCount: 1	ReplicationFactor: 4	Configs: min.insync.replicas=1,confluent.placement.constraints={"observerPromotionPolicy":"leader-is-observer","version":2,"replicas":[{"count":2,"constraints":{"rack":"west"}}],"observers":[{"count":2,"constraints":{"rack":"east"}}]}
+      	Topic: leader-is-observer-promotion	Partition: 0	Leader: 2	Replicas: 2,1,3,4	Isr: 2	Offline: 1	Observers: 3,4
+
+#. Observe the following:
+
+   - In all topics except ``under-min-isr-promotion``, ``multi-region-sync`` and ``under-replicated-promotion``
+     there is only 1 replica in the ISR. This is because replica placement dictated all replicas were in the ``west``
+     region which has only 1 remaining live broker.
+
+   - In the second scenario, the ``multi-region-sync`` topic maintained an ISR of 3 brokers. This is because it's
+     placement policy always allows for brokers from east to join the ISR.
+
+   - The ``under-min-isr-promotion`` and ``under-replicated-promotion`` topics have placement policies that allow
```

Comment:
Should this say _automatically_, to highlight that this happens without manual intervention?

Reply:
fixed